apps/src/applab/designMode.js

```diff
@@ -923,7 +923,7 @@ designMode.onCopyElementToScreen = function(element, destScreen) {
       <b>{elementUtils.getId(duplicateElement)}</b>
     </div>
   );
-  studioApp().displayPlayspaceNotification(alert);
+  studioApp().displayPlayspaceAlert('notification', alert);
```

Comment:
I'm a tad worried about the fragility of using a string literal for `'notification'` since it's also used in an `if` statement inside `displayPlayspaceAlert`. Consider moving it to the apps/src/constants.js file and accessing it from there.